content/_data/upgrades/2-263-2.adoc

```diff
@@ -71,6 +71,21 @@ A total area of 10 million pixels is now allowed by default (e.g. 4000 x 2500),
 // /* package for test */ static /* non-final for script console */ int MAX_AREA = SystemProperties.getInteger(.class.getName() + ".maxArea", 10_000_000); // 4k*2.5k 
 This limit can be changed by setting the link:/doc/book/managing/system-properties/#hudson-util-graph-maxarea[Java system property `hudson.util.Graph.maxArea`] to the desired integer value.
 
+[#SECURITY-2021]
+===== Security fix for agent names
+
+link:/security/advisory/2021-01-13/#SECURITY-2021[SECURITY-2021]
+
+A security fix in Jenkins 2.275 and LTS 2.263.2 disallows agent names that could cause Jenkins to override unrelated `config.xml` files.
+
+Some plugins (like plugin:mesos[Mesos cloud]) allow unsafe characters (like ':') in agent names.
+Those plugins may fail to provision an agent due to the unsafe character in the agent name as reported in jira:JENKINS-65169[].
+
+It is much better to change the agent name in the plugin configuration to avoid unsafe characters.
+If that is not possible, administrators may consider disabling this fix.
+
+Administrators can disable this fix by setting the link:/doc/book/managing/system-properties/#jenkins-model-nodes-enforcenamerestrictions[Java system property `jenkins.model.Nodes.enforceNameRestrictions`] to `false`.
+
```

Comment:
As in an entry further up:

> Doing so is discouraged, as this will re-introduce a security issue.

Reply:
Thanks.  Added in 2b58926469b8673664284925cbbcf52c9d16b5ba